### PR TITLE
Provide command to install automatically all defined project addons

### PR DESCRIPTION
**Pull request by mussbach**
### Ticket
https://www.dev.diplanung.de/DefaultCollection/EfA%20DiPlanung/_workitems/edit/5706

To ease deployment this PR allows to define all addons with their Versions that should be installed and provides a new command to install them e.g. during update or container build

### How to review/test
define addons in projects config/addons.yml and call bin/project dplan:addon:autoinstall

### PR Checklist

Delete the checkbox if it doesn't apply/isn't necessary.

- [x] Tests updated/created
- [ ] Update documentation
- [x] Link all relevant tickets
- [ ] Move the tickets on the board accordingly
- [ ] Data-Cy attributes added/updated ([conventions](https://dplan-documentation.demos-europe.eu/development/guidelines-conventions/coding-styleguides/twig_html.html#guideline-for-naming-cypress-hooks))

### CODE REVIEW

File: config/addons.yml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# example usage
+#dplan_addons:
+#    -
+#      name: "mygithuborga/myaddon"
+#      version: "0.6.1"
```
Comment (MoritzMandler) [inline config/addons.yml:5]: here I would like to add the v in front: ```version: "v0.6.1"```

**Review by MoritzMandler — CHANGES_REQUESTED** [review]
I think this is very comfortable - but I am also a bit afraid that errors that happen during the installation can easily be overlooked.

File: config/addons.yml (hunk 1 above)
Comment (mussbach) [reply config/addons.yml]: indeed, thx [e82c204](https://github.com/demos-europe/demosplan-core/pull/3633/commits/e82c204555a821fc8382c1a3eef6ce45e8b96860)

File: config/addons.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+# example usage
+#dplan_addons:
+#    -
+#      name: "mygithuborga/myaddon"
```
[the remark this answers is not in the document]
Comment (mussbach) [reply config/addons.yml]: good idea [e82c204](https://github.com/demos-europe/demosplan-core/pull/3633/commits/e82c204555a821fc8382c1a3eef6ce45e8b96860)

**Review by MoritzMandler — APPROVED** [review]
lgtm - tested, works fine

**Pull request merged**